admin_manual/configuration_files/files_locking_enabling.rst

```diff
@@ -16,4 +17,28 @@ underlying filesystem.
 The Files Locking app has no configuration options; all you need to do is 
 enable or disable it on your Apps page.
 
-.. figure:: ../images/files_locking_app.png
\ No newline at end of file
+.. figure:: ../images/files_locking_app.png
+
+We recommend using `Redis <http://redis.io/>`_ as your ownCloud memcache when 
+you enable the Files Locking app. Memcached, the popular distributed memory 
```

Comment:
There is a major misunderstanding here.

The files_locking app is an old app that locked on filesystem level. That app does not need memcache at all.

Since 8.1, there is an **experimental** new file locking mechanism, totally unrelated to the files_locking app. That one can be enabled with "filelocking.enabled". That one requires redis.
In the future, if the new file locking mechanism works well, it will obsolete the old files_locking app.